golang/sqlboiler/templates/singleton/schema.tpl

```diff
@@ -213,6 +220,59 @@ input Update{{$modelName}}Input {
 	{{- end }}
 }
 
+input Search{{$modelName}}Args {
+	{{range $column := $table.Columns }}
+	{{- if eq $column.Name "id" }}
+	{{- else if eq $column.Name "created_by" }}
+	{{- else if eq $column.Name "created_at" }}
+	{{- else if eq $column.Name "updated_by" }}
+	{{- else if eq $column.Name "updated_at" }}
+	{{- else if eq $column.Type "[]byte" }}
+	  {{camelCase $column.Name}}: String
+	{{- else if eq $column.Type "bool" }}
+	  {{camelCase $column.Name}}: Boolean
+	{{- else if eq $column.Type "float32" }}
+	  {{camelCase $column.Name}}: Float
+	{{- else if eq $column.Type "float64" }}
+	  {{camelCase $column.Name}}: Float
+	{{- else if eq $column.Type "int" }}
+	  {{camelCase $column.Name}}: Int
+	{{- else if eq $column.Type "int16" }}
+	  {{camelCase $column.Name}}: Int
+	{{- else if eq $column.Type "int64" }}
+	  {{camelCase $column.Name}}: Int64
+	{{- else if eq $column.Type "null.Bool" }}
+	  {{camelCase $column.Name}}: Boolean
+	{{- else if eq $column.Type "null.Byte" }}
+	  {{camelCase $column.Name}}: string
+	{{- else if eq $column.Type "null.Bytes" }}
+	  {{camelCase $column.Name}}: string
+	{{- else if eq $column.Type "null.Float64" }}
+	  {{camelCase $column.Name}}: Float
+	{{- else if eq $column.Type "null.Int" }}
+	  {{camelCase $column.Name}}: Int
+	{{- else if eq $column.Type "null.Int16" }}
+	  {{camelCase $column.Name}}: Int
+	{{- else if eq $column.Type "null.Int64" }}
+	  {{camelCase $column.Name}}: Int64
+	{{- else if eq $column.Type "null.JSON" }}
+	  {{camelCase $column.Name}}: String
+	{{- else if eq $column.Type "null.String" }}
+	  {{camelCase $column.Name}}: String
+	{{- else if eq $column.Type "null.Time" }}
+	  {{camelCase $column.Name}}: Time
+	{{- else if eq $column.Type "string" }}
+	  {{camelCase $column.Name}}: String
+	{{- else if eq $column.Type "time.Time" }}
+	  {{camelCase $column.Name}}: Time
+	{{- else if eq $column.Type "types.Byte" }}
+	  {{camelCase $column.Name}}: String
+	{{- else if eq $column.Type "types.JSON" }}
+	  {{camelCase $column.Name}}: String
```

Comment:
think we need the `!` on these non-null types

Reply:
Moved to https://github.com/choonkeat/hellocrud/pull/5

I put as _optional_ because was thinking it's not required for users to put as search args?